ReleaseNotes.md

```diff
@@ -1,3 +1,3 @@
-# Release 2.10.0
-* Adds Navigation Flight Procedures endpoint.
-* Updates Navigation endpoints with releaseId parameter.
\ No newline at end of file
+# Release 2.11.0
+* Build both netstandard 2.0 and 2.1 clients
```

Comment:
Totally minor nitpick but I would add periods to the ends of these :)

How's THAT for a valuable review?